src/main/resources/application.properties

```diff
@@ -1 +1,11 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/book-store
+spring.datasource.username=root
+spring.datasource.password=Root1234!
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
+spring.jpa.open-in-view=false
 
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+
+spring.jackson.deserialization.fail-on-unknown-properties=true
```

Comment:
why do you need this property?

Reply:
@Parashchuk you didn't answer this question :) why do you need this jackson property?

Reply:
Because, if we put some other properties in the req body but also all the necessary properties, it won't throw a 400 error, but just proceed. It's literally topic from one of the videos in the course in mate academy